index.js

```diff
@@ -55,11 +55,15 @@ module.exports = {
     // Use the super pattern to patch the i18n wrapper function
     // of the apostrophe-templates module
     var superI18n = self.apos.templates.i18n;
-    self.apos.templates.i18n = function(req, name, key) {
+    self.apos.templates.i18n = function(req, name, namespace, key) {
```

Comment:
Do we need to update the docs related to this method? It's currently:
> Takes `req`, `operation` which will be '__', '__mf' or another standard helper name provided by the i18n module, and the arguments intended for that helper, beginning always with `key`. 

Are we still beginning with `key`? I admit I haven't grokked the i18n stuff all that well.

Reply:
We do, but this override is correct, it's the documentation upstream that needs an update to cover that namespace is present if the method being wrapped is one of the new namespaced ones. So that can be a separate issue, on the apostrophe module.